### ISSUE TITLE
ColorWheel: Labelling is not useful for screen reader user.

### ISSUE THREAD

**Issue by jnurthen** [adobe/react-spectrum#4111]
# 📝 Feedback

Labeling the color in the color wheel with the number of degrees is not an equitable experience for screen reader users. I understand that the color wheel is not often used on its own - but there should be a way to read a more useful label than the number of degrees on the color wheel.

## 🌍 Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| [Package name]   | ColorWheel
| Browser          | All
| Operating System | All (w/ screen readers)

**Comment by devongovett** [comment]
The color components are not yet finished from an accessibility point of view, which is why they are still in beta. We are planning on using a list of color names, translated into each locale, to find the closest hue name with modifiers like light/dark etc. This should be much more descriptive and understandable to a user. However, it is currently blocked on figuring out how to distribute these translated strings in a way that doesn't bloat the bundle size (i.e. loading languages on demand). Some prior discussion here: https://github.com/adobe/react-spectrum/pull/1732#issuecomment-818311480.
